blobby.py

```diff
@@ -18,44 +18,82 @@ def validpath(string):
 parser.add_argument("-i", "--image", help="Image to decorate", type=file, required=True)
 parser.add_argument("--show", help="Show image instead of saving to file", action="store_true")
 parser.add_argument("--yell", help="Make old man yelling at X emoji instead of blob", action="store_true")
+parser.add_argument("--cheer", help="Show image of blob cheering instead of normal blob", action="store_true")
 parser.add_argument("-o", "--output", help="Output image (default is ./blob-<image>.png or ./old-man-yells-at-<image>.png)", type=validpath)
 args = parser.parse_args()
 
 # Constants
 IMGSIZE = 170  # Size of final image (square)
 YELL_IMGSIZE = 128  # Size of final image (square)
+CHEER_IMGSIZE = 170  # Size of final image (square)
 SCALAR = 16 # Amount to oversize temporary images so resampling will not show jagginess.  Should be exponential to 4.
 SCRIPT_DIR = os.path.dirname(__file__)
 BLOB_FILE = f"{SCRIPT_DIR}/icons/blob.png"
 YELL_FILE = f"{SCRIPT_DIR}/icons/old-man-yells-at.png"
+CHEER_FILE = f"{SCRIPT_DIR}/icons/blob-cheer.gif"
 LOGO_SIZE = (78,78)
 YELL_LOGO_SIZE = (50,50)
+CHEER_LOGO_SIZE = (60,60)
 FILE_BASE = 'blob-'
 YELL_FILE_BASE = 'old-man-yells-at-'
+CHEER_FILE_BASE = 'blob-cheer-'
 
 if args.yell:
   IMGSIZE = YELL_IMGSIZE
   BLOB_FILE = YELL_FILE
   LOGO_SIZE = YELL_LOGO_SIZE
   FILE_BASE= YELL_FILE_BASE
 
-imgfinal = Image.new("RGBA", (IMGSIZE,IMGSIZE))
-imgbase = Image.open(BLOB_FILE).convert("RGBA")
-imgbase.thumbnail((IMGSIZE, IMGSIZE))
-imgfinal.paste(imgbase, (int((IMGSIZE - imgbase.width)/2), int((IMGSIZE - imgbase.height)/2)), imgbase) # Centered
+if args.cheer:
```

Comment:
We may want to add some validation to ensure that only yell or cheer is provided as an arg (exit failure if both are passed in). Otherwise cheer always takes precedence and it may not be clear to the end user.